src/ddmd/expression.d

```diff
@@ -8772,7 +8773,7 @@ extern (C++) final class DotIdExp : UnaExp
                     if (v.type.ty == Terror)
                         return new ErrorExp();
 
-                    if ((v.storage_class & STCmanifest) && v._init)
+                    if ((v.storage_class & STCmanifest) && v._init && !wantsym)
```

Comment:
>  longer than 80 columns

newline? :)

(the frontend code is impenetrable. Non-descriptive variable and function names is big problem.)